frontend/src/features/admin-form/settings/components/FormEmailSection.tsx

```diff
@@ -50,6 +50,11 @@ const AdminEmailRecipientsInput = ({
 
   const { data: settings } = useAdminFormSettings()
 
+  const requiredAdminEmailValidationRules =
+    useRequiredAdminEmailValidationRules()
+  const optionalAdminEmailValidationRules =
+    useOptionalAdminEmailValidationRules()
```

Comment:
(not a comment, just a thought)

earlier what I had in mind was something like
```
const { t } = useTranslation()
const requiredEmailRules = useMemo(() => 
    createRequiredAdminEmailValidationRules(t), [t])
const optionalEmailRules = useMemo(() => 
    createOptionalAdminEmailValidationRules(t), [t])
```
so that `t` is injected instead at component level and `useTranslation` is only called once, but I think it's prematurely optimising and have imperceptible difference anyway. 

just realised the one you wrote I think works better and more fuss-free, so I'm aligned 🎉